fastlane/swift/SocketClient.swift

```diff
@@ -140,29 +149,33 @@ class SocketClient: NSObject {
     }
 
     private func sendThroughQueue(string: String) {
-        streamQueue.async {
-            let data = string.data(using: .utf8)!
-            _ = data.withUnsafeBytes { self.outputStream.write($0, maxLength: data.count) }
-        }
+        let data = string.data(using: .utf8)!
+        _ = data.withUnsafeBytes { self.outputStream.write($0, maxLength: data.count) }
     }
 
     private func privateSend(string: String) {
-        self.dispatchGroup.enter()
-        sendThroughQueue(string: string)
-
-        let timeoutSeconds = self.cleaningUpAfterDone ? 1 : self.commandTimeoutSeconds
-        let timeToWait = DispatchTimeInterval.seconds(timeoutSeconds)
-        let commandTimeout = DispatchTime.now() + timeToWait
-        let timeoutResult =  self.dispatchGroup.wait(timeout: commandTimeout)
-
-        _ = testDispatchTimeoutResult(timeoutResult, failureMessage: "Ruby process didn't return after: \(SocketClient.connectTimeoutSeconds) seconds", timeToWait: timeToWait)
+        writeQueue.sync {
+            writeSemaphore.wait()
+            print("ESTE")
+            print("Enter \(#function) \(#line)")
+            print(string)
```

Comment:
Probably delete some of these but also...

Either delete or 👇 ?

```suggestion
            verbose(message: "Enter \(#function) \(#line)")
```